aries-test-harness/features/steps/didcomm-v2.py

```diff
@@ -0,0 +1,77 @@
+# -----------------------------------------------------------
+# Behave Step Definitions for DIDComm V2
+#
+# -----------------------------------------------------------
+
+from typing import Optional
+from behave import given, when
+import json, time
+from agent_backchannel_client import (
+    agent_backchannel_GET,
+    agent_backchannel_POST,
+    expected_agent_state,
+    setup_already_connected,
+)
+
+# create invitation: saves invitation object to context.invitation_v2[inviter]
+@when('"{inviter}" creates a didcomm v2 invitation with goal code "{goal_code}"')
+@when('"{inviter}" creates a didcomm v2 invitation')
+def step_impl(context, inviter: str, goal_code: Optional[str] = None):
+    inviter_url = context.config.userdata.get(inviter)
+
+    data = {}
+
+    if goal_code:
+        data["goal-code"] = goal_code
+
+    (resp_status, resp_text) = agent_backchannel_POST(
+        inviter_url + "/agent/command/",
+        "didcomm-v2",
+        operation="create-invitation",
+        data=data,
+    )
+    assert resp_status == 200, f"resp_status {resp_status} is not 200; {resp_text}"
+
+    resp_json = json.loads(resp_text)
+
+    if not hasattr(context, "invitation_v2"):
+        context.invitation_v2 = {}
+
+    context.invitation_v2[inviter] = resp_json["invitation"]
+
+@when('"{accepter}" accepts the didcomm v2 invitation from "{inviter}"')
```

Comment:
The roles here should match what the feature files are stating. Has the role changed to accepter? If so, the feature file needs to change.

Reply:
fixed :+1: